packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx

```diff
@@ -10,17 +10,71 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import React, {
+  createContext,
+  ReactNode,
+  useCallback,
+  useContext,
+  useLayoutEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
 import {
   createTheme,
   ThemeProvider as MuiThemeProvider,
 } from "@mui/material/styles";
-import { memo, ReactNode, useMemo } from "react";
 
 import { ThemeOptions } from "@mui/material";
 import { deepmerge } from "@mui/utils";
 import { createOdysseyMuiTheme, DesignTokensOverride } from "./theme";
 import * as Tokens from "@okta/odyssey-design-tokens";
 import { OdysseyDesignTokensContext } from "./OdysseyDesignTokensContext";
+import { ContrastMode, getBackgroundColor } from "./ThemeContrastProvider";
+
+const useContrastMode = (explicitContrastMode?: ContrastMode) => {
+  const ref = useRef<HTMLDivElement>(null);
```

Comment:
In a future update, we should probably let them pass in their own wrapper. That'll be a separate discussion.